src/index.js

```diff
@@ -50,18 +53,21 @@ class DropdownTreeSelect extends Component {
     inlineSearchInput: PropTypes.bool,
     tabIndex: PropTypes.number,
     disablePoppingOnBackspace: PropTypes.bool,
+    disableKeyboardNavigation: PropTypes.bool,
   }
 
   static defaultProps = {
     onAction: () => {},
     onFocus: () => {},
     onBlur: () => {},
     onChange: () => {},
+    onSearchChange: (_) => {},
```

Comment:
There is an incompatibility between the `prettier` pre-commit hook and the `codeclimate` checks on GitHub with regard to this line. One tool insists on parentheses being present, the other insists it it on being absent. Is there any way we can use the same linter on both so that one tool does not conflict with the other like this?

Reply:
Hmm, not sure. Will have to check with CC docs to see if they support a lint rc.

Reply:
We can ignore the warning manually in CC dashboard as a last resort